NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -267,3 +266,39 @@ def end(self):
         dt = timedelta(seconds=self.__t)
         self.logger.info("total time used by this module is {}".format(dt))
         return dt
+
+
+def calculate_time_shift_for_cosmic_ray(det, station, efield, channel_id):
```

Comment:
Also not sure if it's worth making this function private, I guess we don´t expect it to be used on its own. But this is anyway not something we're very consistent about throughout NuRadio.

Reply:
next PR? Is this function useful for the other efield2Voltage converter modules?

Reply:
Yeah I'm happy for this to be merged as is.